client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/screen-main.tsx

```diff
@@ -134,14 +119,6 @@ const ScreenMain = ( {
 		return name === selectedMainItem;
 	};
 
-	// Set a delay to enable the Continue button since the user might mis-click easily when they go back from another screen
-	useEffect( () => {
```

Comment:
I'm now thinking that we might want this logic in the "Save and continue" button in the Styles screen

Reply:
✅ Done